specification/nginx/resource-manager/readme.md

```diff
@@ -28,7 +28,90 @@ These are the global settings for the Nginx API.
 title: NginxManagementClient
 openapi-type: arm
 openapi-subtype: rpaas
-tag: package-2024-11-01-preview
+tag: package-2025-03-01-preview
+```
+
+### Tag: package-2025-03-01-preview
+
+These settings apply only when `--tag=package-2025-03-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2025-03-01-preview'
+input-file:
+  - NGINX.NGINXPLUS/preview/2025-03-01-preview/openapi.json
+
+suppressions:
+  - code: GetCollectionResponseSchema
+    from: openapi.json
+    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/NGINX.NGINXPLUS/nginxDeployments/{deploymentName}/wafPolicies"]
+    reason: This is by design to avoid high bandwidth consumption as agreed with the partner
+  - code: PutRequestResponseSchemeArm
+    from: openapi.json
+    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/NGINX.NGINXPLUS/nginxDeployments/{nginxDeploymentName}/apiKeys/{apiKeyName}"]
+    reason: This is by design. Request contains a customer provided secret that is not returned in the response. 
+  - code: PutRequestResponseSchemeArm
+    from: openapi.json
+    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/NGINX.NGINXPLUS/nginxDeployments/{nginxDeploymentName}/configurations/{configurationName}"]
+    reason: This is by design. We do not return provided file contents in the response. 
+  - code: PutResponseCodes
+    from: openapi.json
+    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/NGINX.NGINXPLUS/nginxDeployments/{nginxDeploymentName}/apiKeys/{apiKeyName}"]
+    reason: This is a synchronous API, hence we're not returning a 201. 
```

Comment:
That is not actually a valid reason. Synchronous PUT APIs should return 201 when they create a resource

Reply:
Agreed, just checked the HTTP best practices and 201 should be returned when a resource is created. We can fix this violation in the next stable version